models/model_dirac/model_dirac_deltax.cpp

```diff
@@ -19,12 +19,6 @@ status = w_model_dirac(t,x,NULL,user_data);
 
               } break;
 
```

Comment:
I did not change anything in the model. This is what I got out of running the first 3 lines of the example_dirac.